packages/react-core/src/layouts/Flex/Flex.tsx

```diff
@@ -136,6 +136,141 @@ export interface FlexProps extends React.HTMLProps<HTMLDivElement> {
       | 'spaceItems3xl'
       | 'spaceItems4xl';
   };
+  /** Whether to add a gap at various breakpoints. This will override spacers for the main axis. */
```

Comment:
Maybe instead "Gap at various breakpoints. This will override spacers for the main axis." or "Value to add gap at various breakpoints. This will override spacers for the main axis."
When I read "whether", my first thought is it boolean prop.